pkg/cluster/controllers/storage/tenant.go

```diff
@@ -207,6 +223,15 @@ func (m *MinIOController) generateMinIOCR(ctx context.Context, harborcluster *go
 			Image:           image,
 			ImagePullPolicy: m.getImagePullPolicy(ctx, harborcluster),
 			ImagePullSecret: m.getImagePullSecret(ctx, harborcluster),
+			SideCars: &miniov2.SideCars{
+				Containers: []corev1.Container{
+					{
+						Name:    "init-minio",
+						Image:   initImage,
+						Command: []string{initCommand},
+					},
```

Comment:
Does this sidecar support to mount secret? If it supports, let's mount the secret_key to the pod to make it more safer.